src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -303,6 +330,7 @@ export default {
 				this.trackLoans();
 			}
 		});
+		if (this.enableNewLoanCard) this.updateState({ ...this.loanSearchState, pageLimit: 10, pageOffset: 0 });
```

Comment:
Instead of updating the state again, the pageLimit can be set in the data:

```
loanSearchState: { ...getDefaultLoanSearchState(), pageLimit: 10 },
```